sections/05-adc-data-publishing.qmd

```diff
@@ -196,18 +236,32 @@ If you know that you will need to use this process for more than one dataset, we
 
 Once you have finished uploading your data to our servers, please let the Data Team know via email so that we can continue associate your uploaded data with your metadata submission.
 
-As mentioned in [Step 1: The Metadata Submission] section above, when the data package is finalized and made public, there will be a sentence in the abstract that directs users to a separate page where your data will live. The following image is an example of where the data from [this](https://arcticdata.io/catalog/view/doi%3A10.18739%2FA2CV4BS5K) dataset live.
+As mentioned in [Step 1: The Narrative Metadata Submission] section above, when the data package is finalized and made public, there will be a sentence in the abstract that directs users to a separate page where your data will live. The following image is an example of where the data from [this](https://arcticdata.io/catalog/view/doi%3A10.18739%2FA2CV4BS5K) dataset live.
 
 ![](../images/large-data-package.png)
 
-### Step 3: Adding File & Variable Level Metadata
+## Best Practices for Submitting Large Data
 
-The final major section of metadata concerns the structure and content of your data files. Assuming there are many files (and not a few very large ones), it would be unreasonable for users to input file and variable level metadata for each file. When this situation occurs, we encourage users to fill out as much information as possible for each ***unique*** type of file. Once that is completed, usually with some assistance from the Data Team, we will then programmatically carry over the information to other relevant files.
+### Data Organization
 
-When you're data are associated with your metadata submission, they will appear in the data section at the top of the page when you go to edit your dataset. Choose which file you would like to begin editing by selecting the "Describe" button to the right of the file name.
+The data archival process is much smoother when users already have a well structured and organized file management system. As we are at the end of the research life cycle for many users, we understand that you may not already have a well organized file system. In cases such as these, we recommend organizing your data as much as possible ***before*** upload.
```

Comment:
```suggestion
The data archival process is much smoother when users already have a well structured and organized file management system. We strongly recommend starting to work with the Arctic Data Center towards the beginning of your project, instead of the very end. Our curation team can help you publish your data faster, and more smoothly, the earlier you talk to us. Regardless of where you are in your project, however, considering your organization strategy is best done ***before*** data upload.
```